gtsam/geometry/Cal3Fisheye.h

```diff
@@ -160,17 +151,21 @@ class GTSAM_EXPORT Cal3Fisheye {
   /// @name Manifold
   /// @{
 
-  /// Given delta vector, update calibration
-  Cal3Fisheye retract(const Vector& d) const;
-
-  /// Given a different calibration, calculate update to obtain it
-  Vector localCoordinates(const Cal3Fisheye& T2) const;
-
   /// Return dimensions of calibration manifold object
-  virtual size_t dim() const { return 9; }
+  virtual size_t dim() const { return dimension; }
 
   /// Return dimensions of calibration manifold object
-  static size_t Dim() { return 9; }
+  static size_t Dim() { return dimension; }
```

Comment:
THis might be harder to put in base class, with static involved?

Reply:
Yes especially since we are using this to differentiate the various dimensionalities.